modules/virt-using-wasp-agent-to-configure-higher-vm-workload-density.adoc

```diff
@@ -265,18 +265,30 @@ $ oc label namespace wasp openshift.io/cluster-monitoring="true"
 . Enable memory overcommitment in {VirtProductName} by using the web console or the CLI.
 +
 --
-.Web console
-.. In the {product-title} web console, go to *Virtualization* -> *Overview* -> *Settings* -> *General settings* -> *Memory density*. 
-.. Set *Enable memory density* to on.
+* Web console
++
+. In the {product-title} web console, go to *Virtualization* -> *Overview* -> *Settings* -> *General settings* -> *Memory density*. 
+. Set *Enable memory density* to on.
 
-.CLI
-* Run the following command:
+* CLI
++
+. Configure your {VirtProductName} to enable higher memory density and set the overcommit rate:
```

Comment:
```suggestion
* Configure your {VirtProductName} to enable higher memory density and set the overcommit rate:
```
See the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures